ext/uri/tests/rfc3986/modification/userinfo_success_new.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Test Uri\Rfc3986\Uri component modification - userinfo - adding a new one
```

Comment:
It appears this one is mixed up with `ext/uri/tests/rfc3986/modification/userinfo_success_existing.phpt`.